Vector_Class/Vector.h

```diff
@@ -0,0 +1,191 @@
+#ifndef VECTOR_CLASS_VECTOR_H
+#define VECTOR_CLASS_VECTOR_H
+
+#include <iostream>
+
+template <typename T>
+class Vector
+{
+private:
+    size_t size_;
+    size_t capacity_;
+    T* arr = nullptr;
+
+    void relocate()
+    {
+        capacity_ *= 2;
+        T* arr2 = new T[capacity_];
+
+        for(int i = 0; i < size_; ++i)
+            arr2[i] = arr[i];
+
+        delete[] arr;
+        arr = arr2;
+    }
+
+
+public:
+
+    explicit Vector(size_t size = 0, T elem = 0) : size_(size), capacity_(size_)
```

Comment:
do not assign 0 to elem, elem might not a be a number

Reply:
👍